lib/exonio/financial.rb

```diff
@@ -63,7 +63,14 @@ def ipmt(rate, per, nper, pv, fv = 0, end_or_beginning = 0)
     #
     def nper(rate, pmt, pv, fv = 0, end_or_beginning = 0)
       z = pmt * (1 + rate * end_or_beginning) / rate
-      temp = Math.log((-fv + z) / (pv + z))
+
+      y = (-fv + z) / (pv + z)
+
+      if y.negative?
```

Comment:
@cleicar their `.travis.yml` config seems to imply that they support both Ruby 2.1 and Ruby 2.2, and this method was introduced in Ruby 2.3. Maybe consider changing this to `if y > 0`?

2.1 and 2.2 are both unsupported as of June 2018 so maybe they can consider dropping support for those versions.

Reply:
@cleicar oops, meant to say `if y < 0`, sorry about that.